x-pack/legacy/plugins/code/public/components/admin_page/setup_guide.tsx

```diff
@@ -106,8 +106,7 @@ const steps = [
 const toastMessage = (
   <div>
     <p>
-      We’ve made some changes to roles and permissions in Kibana. Read more about what these changes
-      mean for you below.{' '}
+      We’ve made some changes to roles and permissions in Kibana. Read more about how these changes affect your Code implementation below.{' '}
```

Comment:
add an additional `enter` before `{' '}`. or replace it with `&nbsp;&nbsp;`. btw, why do we need it?